pyproject.toml

```diff
@@ -0,0 +1,73 @@
+[build-system]
+requires = ["setuptools >= 61.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "mirdata"
+dynamic = ["version"]
+description = "Common loaders for MIR datasets."
+readme = "README.md"
+license = {file = "LICENSE"}
+requires-python = ">=3.8"
+keywords = ["mir", "dataset", "loader", "music"]
+classifiers = [
+    "License :: OSI Approved :: BSD License",
+    "Programming Language :: Python",
+    "Development Status :: 3 - Alpha",
+    "Intended Audience :: Science/Research",
+    "Topic :: Multimedia :: Sound/Audio :: Analysis",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11"
+]
+dependencies = [
+    "librosa>=0.10.0",
+    "numpy>=1.21.6,<2.0",
+    "pandas>=1.3.5",
+    "tqdm>=4.65.0",
+    "jams>=0.3.4",
+    "py7zr>=0.16.0"
```

Comment:
I can see that several dependencies are removed form this list. I guess the installation still works as expected @tanmayy24?

Reply:
yes, I checked using 
pip install .
pip install ."[tests]"
pip install ."[docs]"

Reply:
Yes tested using
pip install .
pip install ."[tests]"
pip install ."[docs]"